### DISCUSSION TITLE
Citing Quarto

### DISCUSSION THREAD

**Discussion by agrogan1** [quarto-dev/quarto-cli#4366] [Q&A] [answered]
What is the best or preferred way to cite the Quarto software? I did not find this in the quarto.org FAQ.

**Comment by cscheid** [comment, marked as answer]
(Thank you!)

It's in our About page in the [repo](https://github.com/quarto-dev/quarto-cli/), right-hand column:

<img width="459" alt="image" src="https://user-images.githubusercontent.com/285675/219114652-9b85c1af-ba1d-4f8c-84a3-50464f3f4ed0.png">

<img width="550" alt="image" src="https://user-images.githubusercontent.com/285675/219114688-ad024a8b-1bdc-4e3d-9bca-0d1ec818f9de.png">

**Comment by agrogan1** [comment]
Got it! Thank you. I will know to look there in the future. I also figured out that

`library(quarto)`
`citation(package = "quarto")`

will generate the appropriate information for me.

**Comment by arnaudgallou** [comment]
The citation available on the repo doesn't seem to be up to date with the latest release. More generally, is there a way to get the BibTeX citation of the actual version being installed on a machine from the command line? Something that would be roughly equivalent to the following R code:
``` r
toBibtex(citation("pkg"))
```

**Reply by cscheid** [reply]
> The citation available on the repo doesn't seem to be up to date with the latest release.

We'll update this right away and add an item to our checklist so we don't forget to do it in the future

> More generally, is there a way to get the BibTeX citation of the actual version being installed on a machine from the command line?

Not right now, but I quite like that idea. We need to figure out what to do with pre-release version citations, but I'd like to implement something like that.

**Reply by cscheid** [reply]
The citation is up to date now. Can you file a feature request for the bibtex command?

**Reply by arnaudgallou** [reply]
> Can you file a feature request for the bibtex command?

Sure, I'll do that in a few minutes.

**Reply by arnaudgallou** [reply]
For reference #11861.
